src/Shared/FileMatcher.cs

```diff
@@ -2377,15 +2378,15 @@ private static string[] CreateArrayWithSingleItemIfNotExcluded(string filespecUn
 
             if (action == SearchAction.ReturnEmptyList)
             {
-                return ([], action, string.Empty);
+                return ([], action, string.Empty, null);
```

Comment:
if we have this tuple wrapped with a class/structure, we would avoid having these nulls exposed, but let's hear from others @maridematte / @SimaTian / @JanKrivanek

Reply:
`readonly record struct` would be nice, but I do not see it as a big deal

Reply:
I'd like to avoid the `#enable nullable` if possible. If I remember correctly there were a few PRs that were trying to remove those from the code.